diff --git a/score/message_passing/BUILD b/score/message_passing/BUILD
index 078c1abf..31412e8b 100644
--- a/score/message_passing/BUILD
+++ b/score/message_passing/BUILD
@@ -61,6 +61,7 @@ cc_library(
         "//score/mw/com/impl:__subpackages__",
         "@score_logging//score/datarouter:__subpackages__",
         "@score_logging//score/mw/log/detail/data_router:__subpackages__",
+        "//visibility:public",
     ],
     deps = select({
         "@platforms//os:qnx": [":message_passing_qnx_dispatch"],
